security/cmd/node_agent_k8s/main.go

```diff
@@ -28,6 +29,13 @@ import (
 	"istio.io/istio/security/pkg/nodeagent/sds"
 )
 
+const (
+	caProvider  = "CA_PROVIDER"
```

Comment:
Suggest to add some comments on these consts and sort them.

Reply:
done.